swagger/v2/patch_move.yaml

```diff
@@ -0,0 +1,142 @@
+Move:
+  type: object
+  required:
+    - type
+    - attributes
+  properties:
+    id:
+      type: string
+      format: uuid
+      example: f0a91e16-1b0e-4e1f-93fe-319dda9933e6
+      description: The unique identifier (UUID) of this object
+    type:
+      type: string
+      example: moves
+      enum:
+        - moves
+      description: The type of this object - always `moves`
+    attributes:
+      type: object
+      properties:
+        reference:
+          type: string
+          example: TM7B3A2S
+          description: Unique human-readable identifier for the Move
+          readOnly: true
+        status:
+          type: string
+          enum:
+            - proposed
+            - requested
+            - booked
+            - in_transit
+            - cancelled
+            - completed
+          description: Indicates the stage in it's lifecycle that this move is at
+        move_type:
+          type: string
+          enum:
+            - court_appearance
+            - prison_recall
+            - prison_transfer
+            - police_transfer
+            - video_remand_hearing
+            - hospital
+          description: Indicates the type of move, e.g. prison transfer or court appearance
+        move_agreed:
+          oneOf:
+            - type: boolean
+            - type: "null"
+          example: "true"
+          description: Indicates if the moved has been agreed
+        move_agreed_by:
+          oneOf:
+            - type: string
+            - type: "null"
+          example: John Does
+          description: Indicates the name of the person who agreed the move
+        time_due:
+          type: string
+          format: date-time
+          example: "2020-05-17T15:04:07.632Z"
+          description: Time due at the destination, e.g. court appointment time
+        updated_at:
+          type: string
+          format: date-time
+          description: Timestamp of when the move was last created or updated
+          readOnly: true
+        created_at:
+          type: string
+          format: date-time
+          description: Timestamp of when the move was created
+          readOnly: true
+        date:
+          oneOf:
+            - type: string
+              format: date
+            - type: "null"
+          example: "2020-05-17"
+          description:
+            Date on which the move is scheduled (mandatory unless status
+            is proposed)
+        date_from:
+          oneOf:
+            - type: string
+              format: date
+            - type: "null"
+          example: "2020-05-17"
+          description: Start date for potential move (mandatory if status is proposed)
+        date_to:
+          oneOf:
+            - type: string
+              format: date
+            - type: "null"
+          example: "2020-05-17"
```

Comment:
I'd suggest changing this as it is the same as the `date_from` and the description says it should be after...